lib/modules/apostrophe-templates/lib/nunjucks-extensions/component.js

```diff
@@ -0,0 +1,55 @@
+// Implements {{ component 'moduleName:componentName' with dataObject }}
```

Comment:
Is this BC coverage or should the syntax here be `{% %}`?

Reply:
Good catch.